### ISSUE TITLE
Aliases are matched incorrectly

### ISSUE THREAD

**Issue by plemarquand** [s-yadav/relative-to-alias#5]
The following input generates incorrect aliases.

```
./
├── components
│   ├── foo
│   │   └── index.js
│   └── utils
│       └── file.js
└── utils
    └── anotherfile.js
```

`index.js`:
```js
import someutil from '../utils/file';
import anotherutil from '../../utils/anotherfile';
```

Run: `relative-to-alias --src ./ --alias "~/utils" --alias-path ./utils`

The imports in index.js are now incorrect.
`index.js`:
```js
import someutil from 'components/~/utils/file'; // 'utils' folder not at the root is found and aliased.
import anotherutil from '~/utils/anotherfile';
```

**Comment by s-yadav** [comment]
I think it might be due to ~.

Does this also produce wrong transformation.
```
relative-to-alias --src ./ --alias "utils" --alias-path ./utils
```

**Comment by s-yadav** [comment]
Also, is this issue on v1 as well?

**Comment by plemarquand** [comment]
@s-yadav It does seem to be related to `~`, the command you provided works as expected.

Also yes, this was reproduced on 1.1.0 (no luck with 2.0.0, see #4).

**Comment by s-yadav** [comment]
I have found the issue will fix and push soon.

**Comment by s-yadav** [comment]
Pushed the fix. Let me know if it works for you now.

**Comment by plemarquand** [comment]
Closer, `relative-to-alias --src ./ --alias "~/utils" --alias-path ./utils` works, resulting in:

```js
import someutil from '../utils/file';
import anotherutil from '~/utils/anotherfile';
```
However running `relative-to-alias --src ./ --alias "~/components" --alias-path ./components` doesn't make any codemods. I'd expect: 
```js
import someutil from '~/components/utils/file';
```

**Comment by s-yadav** [comment]
By default it excludes the alias directory. You can include it with 
--include-alias-path-directory option.

**Comment by plemarquand** [comment]
@s-yadav ah, that did the trick. Thanks! Verified fixed in 2.0.1
